tools/incremental/incremental.go

```diff
@@ -59,7 +59,7 @@ func extractDockerfileParts(dockerfileContent string) (DockerfileData, error) {
 	for _, line := range lines {
 		trimmed := strings.TrimSpace(line)
 
-		if strings.HasPrefix(trimmed, "COPY") && strings.Contains(trimmed, "--from=builder") {
+		if strings.HasPrefix(trimmed, "COPY") && strings.Contains(trimmed, "--from=builder") && strings.Contains(trimmed, "/workspace/bin/") {
```

Comment:
We use `embed` in go to avoid a lot of problems that are otherwise hard to deal with. Can you please elaborate why exactly we will need this soon?